config.sub

```diff
@@ -89,7 +89,7 @@ while test $# -gt 0 ; do
     - )	# Use stdin as input.
        break ;;
     -* )
-       echo "$me: invalid option $1$help"
+       echo "$me: invalid option $1$help" >&2
```

Comment:
All changes are ok. Even for the PHP-7.1 branch since it's a bugfix. Except this particular file will be overriden each time there is an update from the main upstream file located at http://git.savannah.gnu.org/cgit/config.git/tree/config.sub

Reply:
@petk Ok, I removed this change.

Reply:
@mizunashi-mana, patch for updating the originally targeted `config.sub` file has been also sent to http://lists.gnu.org/archive/html/config-patches/2018-10/msg00004.html so it won't get missed. Thanks...